apigee_devportal_kickstart.install

```diff
@@ -659,17 +659,30 @@ function apigee_devportal_kickstart_update_9001() {
  * Installing ckeditor5, claro, olivero.
  */
 function apigee_devportal_kickstart_update_9002() {
+  $module_handler = \Drupal::service('module_handler');
+  $theme_handler = \Drupal::service('theme_handler');
+  $module_installer = \Drupal::service('module_installer');
+  $theme_installer = \Drupal::service('theme_installer');
+
   // Install admin theme and frontend theme.
-  \Drupal::service('theme_installer')->install(['claro']);
-  \Drupal::service('theme_installer')->install(['olivero']);
+  $theme_installer->install(['claro']);
+  $theme_installer->install(['olivero']);
   // Set the admin theme as claro.Theme seven is deprecated and removed from Drupal 10.
   $config = \Drupal::service('config.factory')->getEditable('system.theme');
   $config->set('admin', 'claro')->save();
   // Uninstall deprecated theme and module.
-  \Drupal::service('theme_installer')->uninstall(['seven', 'bartik']);
-  \Drupal::service('module_installer')->uninstall(['hal', 'color']);
+  foreach($deprecated_modules as $deprecated_module) {
```

Comment:
@bhanu951 , please set value for `$deprecated_modules` and `$deprecated_themes`

Reply:
@divya-intelli Done!

Reply:
@bhanu951 , I am getting error when running the update.

Reply:
@divya-intelli  what error are you getting ?

Reply:
I believe this issue can be reproduced only if you are upgrading your site from `apigee_devportal_kickstart 2.1.0` version and below.

Reply:
Drupal error in log. ![image](https://github.com/apigee/apigee-devportal-kickstart-drupal/assets/75604843/7ff047d6-8a42-4ab3-bbc0-cee04f20fc2e)

Reply:
Oh I see, seems I made error while pasting the code from my local repo. Your suggestions should fix the issue. Please try again with latest commits. Thanks.